trafix-codec/src/message/field/value/aliases.rs

```diff
@@ -0,0 +1,29 @@
+//! Definitions of type aliases used for common FIX field values.
+//!
+//! These aliases provide clearer semantic meaning when working with
+//! strongly typed [`Field`](crate::message::field::Field) variants,
+//! while remaining lightweight wrappers around primitive types.
```

Comment:
Type aliases are not really wrappers, so I would change wording here. They do provide clearer semantic meaning. We might want to really wrap them with a transparent type in the future though, but let's discuss that when the time comes.